.codeclimate.yml

```diff
@@ -5,7 +5,7 @@ engines:
     enabled: true
     config:
       file_extensions: "php"
-      rulesets: "cleancode,codesize,controversial,design,naming,unusedcode,phpmd.xml"
+      rulesets: "cleancode,controversial,design,naming,unusedcode,phpmd.xml"
```

Comment:
Since rulesets are additive, this approach should ensure that the custom configurations are used.